src/components/cylc/table/Table.vue

```diff
@@ -82,70 +84,128 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
               </template>
             </v-select>
           </v-col>
-          <v-col
-            cols="12"
-            md="2">
-            <!-- TODO: we shouldn't need to set the height (px) here, but for some reason the Vuetify
-                       components don't seem to agree on the height here -->
-            <v-btn
-              id="c-table-filter-btn"
-              height="40"
-              block
-              outlined
-              @click="filterTasks"
-            >Filter</v-btn>
-          </v-col>
         </v-row>
       </v-col>
     </v-row>
-    <v-data-table
-      :headers="headers"
-      :items="filteredTasks"
-    >
-      <template
-        slot="headerCell"
-        slot-scope="{ header }"
+    <v-row
+      no-gutters
+      class="flex-grow-1 position-relative"
       >
-        <span
-            class="subheading font-weight-light text-success text--darken-3"
-            v-text="header.text"
-          />
-      </template>
-      <template
-          slot="item"
-          slot-scope="{ item }"
+      <v-col
+        cols="12"
+        class="overflow-y-scroll mh-100 position-relative"
```

Comment:
I don't think this is needed. I tried removing it and the scrollbars appear automatically when the content overflows
```suggestion
        class="mh-100 position-relative"
```